checkInput.ts

```diff
@@ -0,0 +1,35 @@
+let isValid: boolean[] = []
+
+for (let messageIndex = 0; messageIndex < numberOfMessages; messageIndex++) {
+    isValid.push(false)
+}
+
+function checkRowInput (messageIndex: number): void {
+    let rowElements = document.getElementsByName("verification" + messageIndex)
+    let yesButton = rowElements[0] as HTMLInputElement
+    let noButton = rowElements[1] as HTMLInputElement
+    let comment = document.getElementsByName("comment" + messageIndex)[0] as HTMLInputElement
+    
+    if ( yesButton.checked || (noButton.checked && comment.value.length > 0) ) {
+        isValid[messageIndex] = true
+    } else {
+        isValid[messageIndex] = false
+    }
+    checkAllInput()
+}
+
+function checkAllInput (): void {
```

Comment:
check implies a pure function returning a bool.

This is doing a change and setting the UI based on the result. Either rename, or split into two functions